controllers/updateservice_controller.go

```diff
@@ -186,7 +200,7 @@ func (r *UpdateServiceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		reqLogger.Error(err, "Failed to update Status")
 	}
 
-	return ctrl.Result{}, err
+	return ctrl.Result{RequeueAfter: time.Duration(5) * time.Minute}, err
```

Comment:
time.Duration(5) means "5 nanoseconds". did you mean simply `5 * time.Minute` ? End result is the same but with less confusion ;)